Tasks/XamariniOS/xamariniOS.js

```diff
@@ -55,7 +55,7 @@ nugetRunner.exec()
     var mdtoolRunner = new tl.ToolRunner(mdtoolPath);
     mdtoolRunner.arg('--verbose');
     mdtoolRunner.arg('build');
-    mdtoolRunner.arg('--configuration:\'' + configuration + '|' + device + '\'');
+    mdtoolRunner.arg('--configuration:\'' + configuration + '|' + device + '\'', true);
```

Comment:
Passing true will prevent the toolRunner from escaping the argument again which can cause errors if configuration has spaces in it